include/sisl/fds/buffer.hpp

```diff
@@ -33,11 +33,8 @@
 
 namespace sisl {
 struct blob {
-    uint8_t* bytes;
-    uint32_t size;
-
-    blob() : blob{nullptr, 0} {}
-    blob(uint8_t* const b, const uint32_t s) : bytes{b}, size{s} {}
+    std::byte* bytes{nullptr};
```

Comment:
There are several places that are using blob as uint8_t*. Shouldn't this be a major version upgrade?

Reply:
probably, though are we using `master` with the `sisl::blob` anywhere? I don't think OM uses it and 1.3 AM is still on stable right? But you're right, a `10.x` is warranted.

Reply:
Yeah, we are only using it in the 2.0 data side of things.